docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,255 @@
+    6S equivalent power supply.
+
+    By default, Vertiq modules have these values set to protect the module's circuits while operating with the supplied voltage at its maximum rating. In order to protect your power supply and module, 
+    please ensure that your configured voltage limits are set according to the **system's lowest maximum voltage rating**.
+
+* **Volts Limit Starting Voltage** defines the value (in volts) at which the module will begin checking if any limits on regeneration need to be applied. A 0V starting voltage means 
+  that regeneration voltage protection is always enabled. A starting voltage configured just below the Volts Limit enables maximum braking under normal conditions, and enables protection 
+  only during voltage spikes.
+  
+* **Volts Limit** defines the absolute maximum supply voltage the module will apply. In some extreme instances, your module may apply a voltage higher than the configured Volts Limit. In order to handle this properly, ensure that there is a buffer between the absolute maximum allowable system voltage and the value of Volts Limit. 
+  For example, if your system can handle a maximum of 30V, your Volts Limit may be 27V. Never set Volts Limit below the maximum supply voltage. For example, if you're using a 6S 
+  battery with a maximum voltage of 25.2V, never set Volts Limit at or below 25.2V. Violating this rule may cause the motor to run away to its maximum speed.
+
+.. _derates:
+
+Protection with Derates
+===========================
+A derate is a scaling factor that adjusts a module's drive voltage to keep it in a safe operating region based on some operating 
+characteristic of the module, such as speed or temperature. All derate values on Vertiq's modules are represented as float values from [0,1].
+
+Suppose that your :ref:`commanded drive voltage <manual_velocity_control_mechanisms>` is 2V, and the current derate is 0.4. 
+The actual drive voltage applied will be (2V * 0.4) = 0.8V. A basic illustration of derate application is shown below
+
+.. image:: ../_static/manual_images/safety/basic_derate_example.png
+    :align: center
+    :scale: 50%
+
+There are three separate derate values that are internally updated, each linked with a different measured module parameter (e.g. speed and temperature). There is a speed derate used to avoid mechanical failure, 
+a microcontroller temperature based derate to avoid microcontroller overheating, and a coil temperature derate to avoid coil overheating. 
+These three derate values are combined and applied to the target command voltage in order to protect your module. Details of each of these derate values as well as 
+how they are combined are discussed below.
+
+Speed Based Derate
+---------------------
+As mentioned above, your module's speed derate is meant to protect your module against speed runaway. Speed derate is calculated using the module's current speed, 
```

Comment:
Going along with the change above from last time, this shouldn't say its to protect against speed runaway anymore.